<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1318</width>
    <height>491</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1111</width>
      <height>461</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page_1">
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>1031</width>
        <height>431</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QPushButton" name="pushButton_6HWired">
         <property name="minimumSize">
          <size>
           <width>141</width>
           <height>91</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>141</width>
           <height>91</height>
          </size>
         </property>
         <property name="text">
          <string>6H-有线</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="pushButton_6HWireless">
         <property name="minimumSize">
          <size>
           <width>141</width>
           <height>91</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>141</width>
           <height>91</height>
          </size>
         </property>
         <property name="text">
          <string>6H-无线</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="pushButton_7HWired">
         <property name="minimumSize">
          <size>
           <width>141</width>
           <height>91</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>141</width>
           <height>91</height>
          </size>
         </property>
         <property name="text">
          <string>7P-有线</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QPushButton" name="pushButton_7HWireless">
         <property name="minimumSize">
          <size>
           <width>141</width>
           <height>91</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>141</width>
           <height>91</height>
          </size>
         </property>
         <property name="text">
          <string>7P-无线</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="pushButton_7SWired">
         <property name="minimumSize">
          <size>
           <width>141</width>
           <height>91</height>
          </size>
         </property>
         <property name="text">
          <string>7S-有线</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="pushButton_6HWireless_7">
         <property name="minimumSize">
          <size>
           <width>141</width>
           <height>91</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="pushButton_6HWireless_8">
         <property name="minimumSize">
          <size>
           <width>141</width>
           <height>91</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QPushButton" name="pushButton_6HWireless_17">
         <property name="minimumSize">
          <size>
           <width>141</width>
           <height>91</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>141</width>
           <height>91</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QPushButton" name="pushButton_6HWireless_18">
         <property name="minimumSize">
          <size>
           <width>141</width>
           <height>91</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>141</width>
           <height>91</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="pushButton_6HWireless_19">
         <property name="minimumSize">
          <size>
           <width>141</width>
           <height>91</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>141</width>
           <height>91</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="pushButton_6HWireless_20">
         <property name="minimumSize">
          <size>
           <width>141</width>
           <height>91</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>141</width>
           <height>91</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QPushButton" name="pushButton_6HWireless_12">
         <property name="minimumSize">
          <size>
           <width>141</width>
           <height>91</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>141</width>
           <height>91</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="pushButton_6HWireless_22">
         <property name="minimumSize">
          <size>
           <width>141</width>
           <height>91</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>141</width>
           <height>91</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="pushButton_7RWired">
         <property name="minimumSize">
          <size>
           <width>141</width>
           <height>91</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>141</width>
           <height>91</height>
          </size>
         </property>
         <property name="text">
          <string>7R-有线</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="pushButton_6HWireless_26">
         <property name="minimumSize">
          <size>
           <width>141</width>
           <height>91</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>141</width>
           <height>91</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QPushButton" name="pushButton_6HWireless_27">
         <property name="minimumSize">
          <size>
           <width>141</width>
           <height>91</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>141</width>
           <height>91</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2"/>
    <widget class="QWidget" name="page"/>
   </widget>
   <widget class="QLabel" name="label_State">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>50</y>
      <width>151</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 240);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>1140</x>
      <y>10</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>状态</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Return">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>360</y>
      <width>152</width>
      <height>91</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>152</width>
      <height>91</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>152</width>
      <height>91</height>
     </size>
    </property>
    <property name="text">
     <string>返回</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Remide">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>180</y>
      <width>151</width>
      <height>171</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(238, 255, 235);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
